rules/S6994/cfamily/metadata.json

```
@@ -0,0 +1,23 @@
+{
+  "title": "static_assert should be preferred to assert when the argument is a compile-time constant",
```

It should be quoted I believe. See also [S6013](https://sonarsource.github.io/rspec/#/rspec/S6013)
```suggestion
  "title": "\"static_assert\" should be preferred to assert when the argument is a compile-time constant",
```